packages/ui-testable/src/testable.ts

```diff
@@ -34,45 +35,57 @@ const testable =
   // ALWAYS_APPEND_UI_TESTABLE_LOCATORS=1
   // We do this because adding those `data-cid` locators slows things down.
   !process.env.ALWAYS_APPEND_UI_TESTABLE_LOCATORS
-    ? () => (Component) => Component
-    : decorator((ComposedComponent) => {
+    ? () => (Component: any) => Component
+    : decorator((ComposedComponent: any) => {
```

Comment:
you can type this to ComponentClass, e.g.
```
? () => (Component: ComponentClass) => Component
: decorator((ComposedComponent: ComponentClass<any>) => {
```

Reply:
Now that `ui-prop-types` is typed, it doesn't give any more warnings.